src/system/connectivity/wifi.cpp

```diff
@@ -15,13 +40,29 @@ static WiFiManager wm;
  * @param wifi_password  [Optional] WiFi Password as a const char string.
  * @return wl_status_t   WiFi status code.
  */
-wl_status_t WifiInitialize (
+    wl_status_t WiFiConnection::WifiInitialize (
     const char* wifi_ssid,
     const char* wifi_password
     )
 {
     logger << LOG_INFO << "Initializing WiFi provider!" << EndLine;
 
+    WiFiManagerParameter templateID("template_id", "TEMPLATE_ID", "template_id",20);
+    WiFiManagerParameter deviceName("device_name", "DEVICE_NAME", "Device_Name", 20);
+    WiFiManagerParameter authToken("auth_token", "AUTH_TOKEN", "auth_token", 20);
```

Comment:
What means 20? it is string length?
In that case, we should define that on a macro like:
```c
#define WIFI_MANAGER_PARAMETER_STRING_MAX_LENGHT 20
```